kata-deploy/kata-deploy.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: kata-label-node
       containers:
       - name: kube-kata
-        image: katadocker/kata-deploy
+        image: brtknr/kata-deploy:1.7.1-debug
```

Comment:
@brtknr - if you are up for submitting a PR we can land, then better revert s/katadocker/brtknr/

Reply:
Will do.